german/Einladen/Vorstellen/Franken/aus_Wuerzburg.md

```diff
@@ -0,0 +1,2 @@
+
+SIe kommt definitv aus Wuerzburg und ist keine Jura- oder Medizinstudentin. Die Volljaehrigkeit ist gegebenfalls durch Lichtbildausweis zu ueberpruefen. Vorsicht ist die Mutter der Porzellankiste.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.